content/html/concepts/semantic-html/terms/summary/summary.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: '<summary>'
+Description: 'Used as a label for the <details> element and when clicked reveals additional information.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Semantic'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<summary>` semantic HTML element is used as a label for the `<details>` element which is visible no matter what state the `<details>` element is in.
+
+## Syntax
+
+The `<summary>` element includes the summary or caption of the rest of the `<details>` element. The content is always seen whether the `<details>` element is in an open or closed state.
```

Comment:
Similar to what I recommended in the `<details>` entry, we should stress the point here that a `<summary>` element can _only_ be used as the first child-element of a `<details>` element and nowhere else.